packages/core/src/sync-store/index.ts

```diff
@@ -576,28 +592,21 @@ export const createSyncStore = ({
         shouldGetTransactionReceipt,
       );
 
+      const blockSelect: Record<string, any> = {};
+      for (const columnKey of columnAccessProfile.blockInclude) {
+        const columnName = columnKey.replace("block.", "");
+        blockSelect[columnName] =
+          PONDER_SYNC.blocks[columnName as keyof typeof PONDER_SYNC.blocks];
+      }
+
+      blockSelect.timestamp = PONDER_SYNC.blocks.timestamp;
+      blockSelect.number = PONDER_SYNC.blocks.number;
+      blockSelect.hash = PONDER_SYNC.blocks.hash;
+      blockSelect.logsBloom = PONDER_SYNC.blocks.logsBloom;
+      blockSelect.parentHash = PONDER_SYNC.blocks.parentHash;
```

Comment:
Is there a centralized list of which columns are required because they are used internally by Ponder? I think we should update the internal types (`InternalBlock`, etc) to reflect this so we can have some type safety around it.